ouroboros-consensus/src/Ouroboros/Storage/Common.hs

```diff
@@ -20,35 +20,21 @@ module Ouroboros.Storage.Common (
   , BinaryInfo (..)
   ) where
 
-import           Cardano.Binary (ToCBOR (..))
 import           Codec.CBOR.Decoding (Decoder)
 import qualified Codec.CBOR.Decoding as Dec
 import           Codec.CBOR.Encoding (Encoding)
 import qualified Codec.CBOR.Encoding as Enc
 import           Codec.Serialise (Serialise (..))
-import           Data.Hashable (Hashable)
 import           Data.Word
 import           GHC.Generics
 
 import           Cardano.Prelude (NoUnexpectedThunks)
+import           Cardano.Slotting.Slot (EpochNo(..), EpochSize(..))
```

Comment:
```suggestion
import           Cardano.Slotting.Slot (EpochNo (..), EpochSize (..))
```
Nitpicking: next time I touch this file, `stylish-haskell` will add those spaces.